ga4gh/configtest.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
```

Comment:
I'd be happier if we could keep all of the argparse stuff in the cli module. The reasoning for this is that it makes it a lot easier for us to keep our CLI applications consistent if we try to share as much code as possible between then. There's a quite a lot of sharing between the apps at the moment, as we use functions to create command line parsers that have the same arguments and so on.